<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoredImagesDialog</class>
 <widget class="QDialog" name="StoredImagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stored images</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="openFolderButton">
        <property name="toolTip">
         <string>Open the media folder</string>
        </property>
        <property name="text">
         <string>Open &amp;folder</string>
        </property>
        <property name="icon">
         <iconset theme="folder" resource="../breeze-dark-qownnotes.qrc">
          <normaloff>:/icons/breeze-dark-qownnotes/16x16/folder.svg</normaloff>:/icons/breeze-dark-qownnotes/16x16/folder.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="openFileButton">
        <property name="toolTip">
         <string>Open image externally</string>
        </property>
        <property name="text">
         <string>Open image</string>
        </property>
        <property name="icon">
         <iconset theme="mail-attachment" resource="../breeze-qownnotes.qrc">
          <normaloff>:/icons/breeze-qownnotes/16x16/mail-attachment.svg</normaloff>:/icons/breeze-qownnotes/16x16/mail-attachment.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="fileTreeWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="toolTip">
         <string>Only show images that are not used in notes</string>
        </property>
        <property name="text">
         <string>Only show orphaned images</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="insertButton">
        <property name="toolTip">
         <string>Add selected images to the current note</string>
        </property>
        <property name="text">
         <string>Add to current note</string>
        </property>
        <property name="icon">
         <iconset theme="list-add" resource="../breeze-qownnotes.qrc">
          <normaloff>:/icons/breeze-qownnotes/16x16/list-add.svg</normaloff>:/icons/breeze-qownnotes/16x16/list-add.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="deleteButton">
        <property name="toolTip">
         <string>Delete selected images</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset theme="list-remove" resource="../breeze-qownnotes.qrc">
          <normaloff>:/icons/breeze-qownnotes/16x16/list-remove.svg</normaloff>:/icons/breeze-qownnotes/16x16/list-remove.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="LineEdit" name="searchLineEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: none;
	padding: 2px 5px 2px 27px;
	background-image: url(:/images/search-notes.svg);
	background-repeat: no-repeat;
	background-position: center left;
	margin-right: 0px;
}
</string>
        </property>
        <property name="placeholderText">
         <string>Find image</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh" resource="../breeze-qownnotes.qrc">
          <normaloff>:/icons/breeze-qownnotes/16x16/view-refresh.svg</normaloff>:/icons/breeze-qownnotes/16x16/view-refresh.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QFrame" name="notesFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Used in notes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="noteTreeWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">Note</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../breeze-dark-qownnotes.qrc"/>
  <include location="../breeze-qownnotes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StoredImagesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StoredImagesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
